toolchains/xbox/CMakeGDKScarlett.cmake

```diff
@@ -0,0 +1,244 @@
+# Adapted by Sentry from:
+# https://github.com/microsoft/Xbox-GDK-Samples/blob/e5328b9c06443739ec9c7c0089a36c5743c9da15/Samples/Tools/CMakeExample/CMake/CMakeGDKScarlett.cmake
+#
+# CMakeGDKScarlett.cmake : CMake definitions for Microsoft GDK targeting Xbox Series X|S
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+mark_as_advanced(CMAKE_TOOLCHAIN_FILE)
+
+if(_GDK_SCARLETT_TOOLCHAIN_)
+    return()
+endif()
+
+set(XBOX_CONSOLE_TARGET "scarlett" CACHE STRING "")
+
+#--- Microsoft Game Development Kit
+set(XdkEditionTarget "240603" CACHE STRING "Microsoft GDK Edition")
+
+message("XdkEditionTarget = ${XdkEditionTarget}")
+
+set(CMAKE_TRY_COMPILE_PLATFORM_VARIABLES XdkEditionTarget BUILD_USING_BWOI)
+
+#--- Windows SDK
+set(SDKVersion 10.0.19041.0)
```

Comment:
According to the GDK documentation, Windows SDK version 10.0.22000+ is required as of October 2023 GDK: https://learn.microsoft.com/en-us/gaming/gdk/_content/gc/getstarted/overviews/sdk-and-tools#windows-software-development-kit-sdk

Reply:
We have a follow up change that will auto detect the latest GDK and Windows SDK versions. I believe its just pending the merge of this change

Reply:
I'm more surprised this even builds with what should be the "wrong" version of the Windows SDK. Thanks for the heads up about the follow-up change!